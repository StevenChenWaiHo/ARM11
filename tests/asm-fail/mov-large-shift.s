mov r11,r2, asr #222
