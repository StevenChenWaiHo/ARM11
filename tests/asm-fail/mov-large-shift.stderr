tests/asm-fail/mov-large-shift.s:1:21: Const too large for a shift const: 222 (max is 31)
